src/parking/controls/editor/editor-form.ts

```diff
@@ -22,7 +22,9 @@ export function getLaneEditForm(osm: OsmWay, waysInRelation: WaysInRelation, cut
                     type="button"
                     class="editor-form__cut-button"
                     style="${waysInRelation[osm.id] ? displayNone : null}"
-                    onclick=${() => cutLaneListener(osm)}>
+                    onclick=${() => {
+        cutLaneListener(osm)
+    }}>
```

Comment:
Add please rule `'@typescript-eslint/no-confusing-void-expression': ['error', { ignoreArrowShorthand: true, ignoreVoidOperator: false }],` for ignore such cases

Reply:
Ah, sorry for not following up on this, yet…